.github/workflows/labeler.yml

```diff
@@ -3,10 +3,15 @@ on:
   pull_request_target:
     types: [closed]
 
+permissions: {}
+
 jobs:
   label:
     name: Labeler
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
Is this the only line that's needed now that we've changed the default to read only?

Reply:
It is also considered a good practice to keep the minimal permission set on the workflow:

1. It provides transparency in the permissions used since the settings are not public
2. It works as a double prevention by preventing from insider risks (since users with admin access can revert the default permissions)

I personally would recommend to keep the minimal permission set on the workflow files either, but if you rather remove them I'd say that the only ones that should stay would be `labeler.yml` and `ci.yml`.

PS: ci.yml does not work only with contents and package read.